airflow/providers/openmldb/provider.yaml

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-openmldb
+name: OpenMLDB
+description: |
+    `OpenMLDB <https://openmldb.ai/>`__
+
+versions:
+  - 1.0.0
+
+dependencies:
+  - apache-airflow>=2.2.0
+  - apache-airflow-providers-http
```

Comment:
Also I thiink @vagetablechicken - you should consider releasing your provider on your own first. 

We had a long discussion recently on that very subject and I would like to see a discussion whether you considered releasing it as your own provider.  I am not sure if we want to take over the mainteance of OpenMLDB provider in the community.

Before we accept any new provider, there should be a discussion at the devlist where you will refer to the discussion you have and justify that it is better to have the provider in Airlfow rather than relase on your own.

See discussion here:

https://lists.apache.org/thread/6ngq79df7op541gfwntspdtsvzlv1cr6

There are certain limitiations and obligations of those who submit providers to be "community" providers rather than providers managed outside of Airlfow Community. If you release a provider on your own, you can fully control your release schedule and decide on how an when you release - so for product like open MLDB it seems more appropriate to release their provider on their own, unless there is a goo reason to get it in.

You can read about the release process and some of the  approach we have for accepting the providers in https://github.com/apache/airflow#release-process-for-providers

I'd strongly encourage you to release the provider on your own first, and only approach contirbution to Airflow when it proves to be popular and useful.

You can add link to the provider of yours at https://airflow.apache.org/ecosystem/ and there are airlfow provider registries available that you can register it too.

If you really want to submit the provider to the community - please start discussion at the devlist, stating why you thnk is good, who are the stakeholders behind it, who from the stakeholder will be maintaing it and what kind of involvement they have and what kind of commitment the stakeholders can make for the new provider.